server/storage/endpoint/gc_key_space.go

```diff
@@ -38,9 +33,10 @@ type KeySpaceGCSafePoint struct {
 // KeySpaceGCSafePointStorage defines the storage operations on KeySpaces' safe points
 type KeySpaceGCSafePointStorage interface {
 	// Service safe point interfaces.
-	SaveServiceSafePoint(spaceID string, ssp *ServiceSafePoint) error
+	// NOTE: field ServiceSafePoint.ExpiredAt will be ignored, use etcd's lease to manage lifetime instead.
```

Comment:
Do we need a deprecated flag for this field to ensure no one will use it anymore?

Reply:
Probably not, we should probably still store when safepoint will be expired, so if and when client need this information, we can retrieve directly from ServiceSafePoint, instead of looking it up by lease.  

`ExpiredAt` field will be calculated at Service level, using `TTL` client provided with server's `time.now`, here we just simply store it. 

I'll modify the comment to make it more clear